data/raw/korea/KOR_policy_data_sources.csv

```diff
@@ -1,47 +1,67 @@
+KOR,Gangwon-do,event_cancel,Public events held by the local government are canceled,N,2020-02-22,1,,,https://news.v.daum.net/v/20200222153614198,04/07/2020,
+KOR,Chungcheongbuk-do,event_cancel,Public events held by the local government are canceled,N,2020-02-23,1,,,http://www.cbinews.co.kr/news/articleView.html?idxno=130368,04/07/2020,
+KOR,Chungcheongnam-do,event_cancel,Public events held by the local government are canceled,N,2020-02-07,1,,,https://www.yna.co.kr/view/AKR20200204145700063,04/07/2020,
+KOR,Sejong,event_cancel,Public events held by the local government are canceled,N,2020-01-31,1,,,http://www.dtnews24.com/news/articleView.html?idxno=570283,04/07/2020,
+KOR,Daejeon,event_cancel,Public events held by the local government are canceled,N,2020-02-24,1,,,https://www.daejeon.go.kr/drh/drhStoryDaejeonView.do?boardId=blog_0001&categorySeq=291&menuSeq=1624&ntatcSeq=1329554704,04/07/2020,
+KOR,Gyeongsangbuk-do,event_cancel,Public events held by the local government are canceled,N,2020-02-04,1,,,https://www.yeongnam.com/web/view.php?key=20200204010000591,04/07/2020,
+KOR,Gyeongsangnam-do,event_cancel,Public events held by the local government are canceled,N,2020-02-05,1,,,https://www.news1.kr/election613/news/articles/?3833871,04/07/2020,
+KOR,Jeju,event_cancel,Public events held by the local government are canceled,N,2020-02-23,1,,,https://www.newsjeju.net/news/articleView.html?idxno=340284,04/07/2020,
+KOR,Gyeonggi-do,event_cancel,Public events held by the local government are canceled,N,2020-01-30,1,,,https://www.open.go.kr/pa/infoWonmun/cateSearch/wonmunOrginlDetail.do?prdnDt=20200130085100&prdnNstRgstNo=DCTEBB8A7416FEBB8E36BFFFFF8FF1EF249#hnp=0.09478994880959113,04/07/2020,
+KOR,Ulsan,event_cancel,Public events held by the local government are canceled,N,2020-02-23,1,,,https://www.ulsanpress.net/news/articleView.html?idxno=345623,04/07/2020,
+KOR,Gwangju,event_cancel,Public events held by the local government are canceled,N,2020-02-06,1,,,https://webcache.googleusercontent.com/search?q=cache:v4o45uHp3IQJ:https://www.gwangju.go.kr/c19/c19/contentsView.do%3FpageId%3Dcoronagj17+&cd=2&hl=en&ct=clnk&gl=us,04/07/2020,
+KOR,Busan,event_cancel,Public events held by the local government are canceled,N,2020-02-08,1,,,http://www.busan.go.kr/youth/gnsaler01/1419512,04/07/2020,
+KOR,Incheon,event_cancel,Public events held by the local government are canceled,N,2020-02-24,1,,,http://www.dtoday.co.kr/news/articleView.html?idxno=350607,04/07/2020,
+KOR,Jeollanam-do,event_cancel,Public events held by the local government are canceled,N,2020-02-07,1,,,https://www.yna.co.kr/view/AKR20200203114000054,04/07/2020,
+KOR,Jeollabuk-do,event_cancel,Public events held by the local government are canceled,N,2020-02-24,1,,,http://www.jeonbuk.go.kr/board/view.jeonbuk?boardId=BBS_0000106&menuCd=DOM_000000105010007000&paging=ok&startPage=1&dataSid=386610,04/07/2020,
```

Comment:
do we want to encode all the `event_cancel` where local gov events are canceled with `policy_intensity<1`?